L1Trigger/L1TTrackMatch/plugins/L1TkMuonProducer.cc

```diff
@@ -468,11 +490,26 @@ void L1TkMuonProducer::runOnMTFCollection_v1(const edm::Handle<RegionalMuonCandB
 
         TkMuon l1tkmu(l1tkp4, l1muRef, l1tkPtr, trkisol);
 
+        if (useRegionEtaMatching_) {
```

Comment:
this exact pattern appears twice in the PR: it could be made into a function like
```cpp
bool inRegionEta(const TkMuon& l1tkmu, int detector) {
  return ((detector == barrel_MTF_region and std::abs(l1tkmu.eta()) <= etaBO_) or
          (detector == overlap_MTF_region and (std::abs(l1tkmu.eta()) >= etaBO_ and std::abs(l1tkmu.eta()) <= etaOE_)) or
          (detector == endcap_MTF_region and std::abs(l1tkmu.eta()) >= etaOE_));
}
```
then here and below would just be:
```cpp
if (useRegionEtaMatching_ and !inRegionEta(l1tkmu, detector))
  continue;
```